src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/ExtendedHearingRequestResponse.java

```diff
@@ -256,6 +257,7 @@ private List<JudicialResultEntity> buildJudicialResults(List<JudicialResult> jud
                             .isConvictedResult(judicialResult.isConvictedResult())
                             .label(judicialResult.getLabel())
                             .judicialResultTypeId(judicialResult.getJudicialResultTypeId())
+                            .resultText(judicialResult.getResultText())
```

Comment:
@rschallapalli  This change will return the result text for the GET.

Reply:
Yes, right, my bad